account-management-app/src/main/java/org/duracloud/account/app/controller/AbstractController.java

```diff
@@ -136,4 +176,32 @@ protected void setFailureFeedback(String message, RedirectAttributes redirectAtt
         UserFeedbackUtil.addFailureFlash(message, redirectAttributes);
     }
 
+    protected boolean validateRecaptcha(final String recaptchaResponse) throws IOException {
+        log.info("recaptcha response from form = {}", recaptchaResponse);
+        boolean success = false;
+        final HttpPost httpPost = new HttpPost("https://www.google.com/recaptcha/api/siteverify");
+        final List<NameValuePair> params = new ArrayList<NameValuePair>();
+        params.add(new BasicNameValuePair("response", recaptchaResponse));
+        params.add(new BasicNameValuePair("secret", getRecaptchSecret()));
+        httpPost.setEntity(new UrlEncodedFormEntity(params));
+
+        try (CloseableHttpClient client = HttpClients.createDefault();
+             CloseableHttpResponse response = (CloseableHttpResponse) client
+                     .execute(httpPost)) {
+            var statusLine = response.getStatusLine();
+            log.info("http status from recaptcha = {}", statusLine.toString());
+            var jsonStr = IOUtils.toString(response.getEntity().getContent(), Charset.defaultCharset());
+            log.info("http response body from recaptcha = {}", jsonStr);
+            final int statusCode = statusLine.getStatusCode();
+            if (statusCode == HttpStatus.SC_OK) {
+                var map = new ObjectMapper().readValue(jsonStr, Map.class);
+                success = (boolean) map.get("success");
```

Comment:
If the response comes back with success == true, then we can proceed to process the form.